app/models/solidus_bolt/payment_method.rb

```diff
@@ -13,5 +13,19 @@ def payment_source_class
     def partial_name
       'bolt'
     end
+
+    def publishable_key
+      bolt_configuration.publishable_key
```

Comment:
[Spree::Config.static_model_preferences](https://github.com/nebulab/solidus_bolt/blob/master/lib/solidus_bolt/engine.rb#L14-L23) allows specifying more than one configuration. 

Instead of overriding these, I think would be better to add it in a static_model_preferences

Reply:
But the [BoltConfiguration](https://github.com/nebulab/solidus_bolt/blob/master/app/models/solidus_bolt/bolt_configuration.rb#L17) itself assumes that there will ever be only one configuration at a time, so that's what I was basing my code on

Reply:
We'll have one BoltConfiguration that should use to set a global setting for bolt and provide a basic auth for all the API call. But the payment method should be able to override that credentials and provide a new one. This is because we should allow the developer to add more than one payment method even for bolt.